### PR TITLE
Cleanups for SplitEditorAreaPane context menu management.

### PR DESCRIPTION
**Pull request by mdickinson**
Two parts to this PR:

- clean up the context menu generation code, fixing some minor style problems
- add fix for the case where `get_context_menu` returns `None`, and make it explicit that `get_context_menu` can return `None`

### CODE REVIEW

File: pyface/ui/qt4/tasks/split_editor_area_pane.py [hunk 1]
```diff
@@ -173,38 +173,50 @@ def set_layout(self, layout):
...
-            actions = [Action(id='split_hor', name='Create new pane to the right',
-                       on_perform=lambda : splitter.split(orientation=
-                        QtCore.Qt.Horizontal)),
-                       Action(id='split_ver', name='Create new pane to the bottom',
-                       on_perform=lambda : splitter.split(orientation=
-                        QtCore.Qt.Vertical))]
-
-            splitgroup = Group(*actions, id='split')
-            menu.append(splitgroup)
+            split_group = Group(
```
Comment (mdickinson) [inline pyface/ui/qt4/tasks/split_editor_area_pane.py]: Changes in this block are purely stylistic, except that I've changed "Create new pane to the bottom" to the more natural sounding (at least to me) "Create new pane below".

File: pyface/ui/qt4/tasks/tests/test_split_editor_area_pane.py [hunk 2]
```diff
@@ -382,6 +382,35 @@ def test_context_menu_merge_text_top_bottom_split(self):
...
+        tab_widget, = editor_area_widget.tabwidgets()
+
+        context_menu_event = QtGui.QContextMenuEvent(
+            QtGui.QContextMenuEvent.Mouse,
+            # Position relative to the widget, so (-1, -1) should be
+            # reliably outside.
+            QtCore.QPoint(-1, -1),
+        )
+
+        # Just exercise the context menu code to make sure it doesn't raise.
```
Comment (mdickinson) [inline pyface/ui/qt4/tasks/tests/test_split_editor_area_pane.py]: This could also do with a test to check directly that `get_context_menu` is returning `None`. I'll add that.

Comment (mdickinson) [reply pyface/ui/qt4/tasks/tests/test_split_editor_area_pane.py]: Done.
